.gitattributes

```diff
@@ -0,0 +1 @@
+*.sh eol=lf
```

Comment:
I've added .gitattributes here as checking out this repository in Windows causes the Dockerfile not to build. By allowing the end-of-line to use just line-feeds (linux style,) these binaries can properly run in the docker image.